### PR TITLE
Docs 1066 Scale Intro

### PR DESCRIPTION
**Pull request by q5sys**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/en/hub/scale/_index.md [hunk 1]
```diff
@@ -5,3 +5,6 @@ description: "Articles about the in-development TrueNAS SCALE project"
 weight: 6
 ---
 
+TrueNAS SCALE is a free and Open Source Hyperconverged Infrastructure (HCI) solution. Built on TrueNAS CORE, SCALE adds Linux Containers, KVM, and scale-out ZFS storage capabilities.
```
Comment (iXTimExample) [inline content/en/hub/scale/_index.md:8]: Lets linkify "TrueNAS SCALE" to the updated overview page

**Review by BonnieF — CHANGES_REQUESTED** [review]
Just need a period after:
Articles about the in-development TrueNAS SCALE project

**Comment by q5sys** [comment]
> Just need a period after:
> Articles about the in-development TrueNAS SCALE project

@Mrt134  Are we putting periods in titles and descriptions now?  I missed memo that if we are.

**Comment by iXTimExample** [comment]
When we're writing a full sentence in the description, we should end with a period.

**Comment by q5sys** [comment]
ready for re-review

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**